internal/persistence/client.go

```diff
@@ -0,0 +1,59 @@
+package persistence
+
+import (
+	"fmt"
+
+	"github.com/boltdb/bolt"
+	"github.com/madjlzz/madprobe/internal"
+	"github.com/madjlzz/madprobe/internal/model"
+)
+
+const (
+	probeBucket = "probe"
+)
+
+type PersistenceClient struct {
+	probes map[string]*model.Probe
```

Comment:
I was for removing totally the `map`. As we have now a database, the `map` will act more as a `cache`. I want to keep things as simple as possible for the moment and caching may be implemented more riguour later.
To conclude, every time we retrieve probes, we ask the db to return values. (we will still need to keep a reference to launched probe to stop them if client asks)

Reply:
I understand, but I think we might still need a map to handle the Finish channel in probe if we want to stop them. We could replace this map with a `map[string]chan bool`for this matter, and ask the db for probe info.

Reply:
Yup, go that way please 🥇